internal/idle/idle.go

```diff
@@ -182,6 +182,7 @@ func (m *Manager) tryEnterIdleMode() bool {
 	return true
 }
 
+// EnterIdleModeForTesting instructs the channel to enter idle mode.
```

Comment:
Very optional: 

Looking at https://pkg.go.dev/google.golang.org/grpc/internal/idle#Manager.EnterIdleModeForTesting, should we add more info from `tryEnterIdleMode()` here?